routes/soldiers.js

```diff
@@ -0,0 +1,65 @@
+import { addNewSoldier, lookForSoldier, lookForAllSoldiers } from '../database.js';
+
+const newSoldierSchema = {
+  body: {
+    type: 'object',
+    required: ['_id', 'name', 'rank', 'limitations'],
+    properties: {
+      _id: { type: 'integer' },
```

Comment:
i think it is `id` in the spec... dont mess with the PO :)

Reply:
fixed